gordo_components/cli/cli.py

```diff
@@ -228,15 +282,17 @@ def build(
                 output_dir,
                 model_register_dir,
                 replace_cache=True,
+                cv_mode=cv_mode,
             )
             saved_metadata = load_metadata(model_location)
             all_scores = get_all_score_strings(saved_metadata)
 
         for score in all_scores:
             print(score)
 
-    # Write out the model location to this file.
-    model_location_file.write(model_location)
+    # Write out the model location to this file if it exists and cv_mode is not equal to cross_val_only.
+    if cv_mode != "cross_val_only":
+        model_location_file.write(model_location)
```

Comment:
What do you think about doing a simple `return` after you've done what you need to do in `cross_val_only` condition, and then remove this `if`.
Alternatively, still remove the `if` but indent this into the `else` block.